package-lock.json

```diff
@@ -8,7 +8,7 @@
 			"name": "visual-studio-code-plugin-for-alm-octane",
 			"version": "2025.2.0",
 			"dependencies": {
-				"@microfocus/alm-octane-js-rest-sdk": "^25.1.0",
+				"@microfocus/alm-octane-js-rest-sdk": "file:../alm-octane-js-rest-sdk",
```

Comment:
the changes that I was talking about before are still here. you only have to modify the package.json and then run npm install. we never manually modify the -lock file